### PR TITLE
Beta can sig

### PR DESCRIPTION
**Pull request by FlorianBarkmann**

### CODE REVIEW

File: cansig/integration/training/training_plan.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+            return beta
+        return beta / stage * (step % period)
+
+    return beta
+
+
+# Ideally, we would inherit from scVI's trainingsplan, however, it calls the kl_weight
+# in the initializer and since CanSigTrainingPlan is not yet initialized it doesn't find
+# the property. There is probably a way of solving it, but I don't know how.
+class CanSigTrainingPlan(pl.LightningModule):
```
Comment (FlorianBarkmann) [inline cansig/integration/training/training_plan.py:69]: @pawel-czyz do you know how to solve this?

Comment (pawel-czyz) [reply cansig/integration/training/training_plan.py:69]: Somewhat hacky, but how about this: inherit, but implement a custom constructor? You don't need to use `super().__init__(...)`.

File: cansig/integration/training/trainer.py [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
 class UnsupervisedTrainingCanSig:
     """General purpose unsupervised train method."""
 
     module_batch_effect: CanSigBaseModule
 
     def train(
         self,
-        max_epochs: Optional[int] = None,
+        max_epochs: int = 400,
+        cnv_max_epochs=400,
```
Comment (pawel-czyz) [inline cansig/integration/training/trainer.py:23]: Nit: missing type annotations.

File: setup.cfg [hunk 3]
```diff
@@ -18,6 +18,7 @@ packages =
     cansig.models
     cansig.plotting
     cansig.run
+    cansig.integration
```
Comment (pawel-czyz) [inline setup.cfg:21]: Is this actually needed? I thought we discover the packages automatically since I moved the code to `src`.

**Pull request closed without merging**